src/functions/marshal_headers_from_sapi.php

```diff
@@ -17,6 +17,19 @@
  */
 function marshalHeadersFromSapi(array $server) : array
 {
+    $contentHeaderLookup = isset($server['LAMINAS_DIACTOROS_STRICT_CONTENT_HEADER_LOOKUP'])
+        ? function (string $key) : bool {
```

Comment:
the function could be static and thus static variable could be omitted?

Reply:
I tried and got a linting error for the function definition, indicating static values cannot be dynamic. (I thought that would work, too, and had done it in the initial attempt.)

Reply:
PHP kills me sometimes. I can do it fine as part of the ternary, but when I was defining the functions as separate static variables previously, I was getting the linting error. Go figure.

Reply:
Also, static variables within the closure will help with performance as well, as they don't then need to be redefined on each invocation.